source/api/image/3.0/change-log.md

```diff
@@ -0,0 +1,158 @@
+The `attribution` property could not specify the label to render with the value, and thus clients typically used "Attribution". This was not able to be internationalized, nor changed in contexts where "Attribution" has specific meaning (such as for artworks, where the attribution is assignment of the creator). The property was renamed to `requiredStatement` to remove the rights-specific semantics, and the structure was changed to allow both `label` and `value` to allow for appropriate labeling with internationlization. See issue [#1287](https://github.com/IIIF/api/issues/1287).
+
+Additionally, in the [Presentation API][prezi3] and here, a new pattern has been adopted for all textual values of a JSON object with the language code as the key (or `@none` if the language is not known) and the content as a string within an array as the value. This applies only to the `requiredStatement` propery in the Image API. The pattern relies on features that have been introduced by the JSON-LD Community Group, and are not yet standardized. See issue [#755](https://github.com/IIIF/api/issues/755).
+
+#### 1.2.3. Rename `license` to `rights`, allow only a single value
+
+The `license` property was renamed to `rights` to accomodate both rights statements and usage licenses. The value is constrained to be a single URI, an array is no longer permitterd. The value is further constrained to allow only Creative Commons URIs, RightsStatements.org URIs and URIs registered as extensions. This additional constraint is to allow clients to treat the property as an enumeration rather than free text, and implement URI specific behavior. See issues [#644](https://github.com/IIIF/api/issues/644) and [#1479](https://github.com/IIIF/api/issues/1479).
+
+#### 1.2.4. Require an array of JSON objects for the `logo` property
+
+Previous versions of the specification allowed the `logo` property have a single value or an array value, each with either string or JSON object contents. The value of the `logo` property must now always be an array of JSON objects, even if there is only one object. This reduces the type checking needed on values by clients as they can always iterate over the array, improving the developer experience. See issue [#1131](https://github.com/IIIF/api/issues/1131).
+
+#### 1.2.5. The `type` property is required on all resources, with new values
+
+The `type` property with a single value is now required on all resources, including content resources and services. This serves several purposes, including facilitating object mapping code libraries, and forcing the serialization to generate a JSON object for the resource, not just a string with the resource's URI. The values for `type` have been changed version-specific strings that avoid the namespace structure, for example from `iiif:Image` in 2.1 to `ImageService3` in 3.0. Note that the `@type` property is used only when referring to object from older specifications such as the Authentication API 1.0. See issue []().
```

Comment:
Missing issue.

Reply:
... have been changed *to* version-specific ...

Reply:
There is no issue (not in prezi log either), fixed missing **to**